aiaccel/cli/set_result.py

```diff
@@ -45,13 +47,13 @@ def main() -> None:
         parameters_config = HyperParameterConfiguration(config.optimize.parameters)
 
         for p in parameters_config.get_parameter_list():
-            if p.type.lower() == "float":
+            if p.type.lower() == data_type_uniform_float:
```

Comment:
書き方の問題だとは思うんですが，
```suggestion
            if data_type.is_uniform_float(p.type):
```
みたいな方がきれいだと思うんですが如何でしょう？